src/vs/editor/test/common/modes/textToHtmlTokenizer.test.ts

```diff
@@ -293,7 +293,7 @@ suite('Editor Modes - textToHtmlTokenizer', () => {
 
 class Mode extends Disposable {
 
-	private readonly languageId = 'textToHtmlTokenizerMode';
+	public readonly languageId = 'textToHtmlTokenizerMode';
```

Comment:
Is this also part of fixing the types, or is it a separate issue?

Reply:
This is actually one of the bugs revealed by this change :) Lots of code in this file was accessing `.languageId` even though it was private